src/Magnum/ImGuiIntegration/Context.cpp

```diff
@@ -130,6 +130,8 @@ Context::Context(NoCreateT) noexcept: _context{nullptr}, _shader{NoCreate}, _tex
 
 Context::Context(Context&& other) noexcept: _context{other._context}, _shader{std::move(other._shader)}, _texture{std::move(other._texture)}, _vertexBuffer{std::move(other._vertexBuffer)}, _indexBuffer{std::move(other._indexBuffer)}, _timeline{std::move(other._timeline)}, _mesh{std::move(other._mesh)}, _supersamplingRatio{other._supersamplingRatio}, _eventScaling{other._eventScaling} {
     other._context = nullptr;
+    /* Update the pointer to _texture */
+    ImGui::GetIO().Fonts->SetTexID(reinterpret_cast<ImTextureID>(&_texture));
```

Comment:
Maybe this should do `ImGui::SetCurrentContext(_context);` first, to ensure we don't update some other context.

Reply:
And then set it back later?